docs/ssl.md

```diff
@@ -16,25 +16,25 @@ Once you update the listen and try to run verdaccio again will ask for certifica
 * Generate your certificates
 
 ````
- $ openssl genrsa -out ~/.config/verdaccio/verdaccio-key.pem 2048
- $ openssl req -new -sha256 -key ~/.config/verdaccio/verdaccio-key.pem -out ~/.config/verdaccio/verdaccio-csr.pem
- $ openssl x509 -req -in ~/.config/verdaccio/verdaccio-csr.pem -signkey ~/.config/verdaccio/verdaccio-key.pem -out ~/.config/verdaccio/verdaccio-cert.pem
+ $ openssl genrsa -out /Users/user/.config/verdaccio/verdaccio-key.pem 2048
```

Comment:
That looks a bit macOS-specific to me? Why did you remove the tilde which should be working fine on Linux and macOS?

Reply:
It was misleading, somehow only resolves absolute paths, I tried with the tilde and did not work. I put in in my todo, but in order to not confuse users, it has to be an absolute path.

Reply:
Sure, no problem :) I just wanted to make sure it was not changed to this by accident